game/webserver.cc

```diff
@@ -203,6 +218,46 @@ http_server::response WebServer::POSTresponse(const http_server::request &reques
 	}
 }
 
+std::map<std::string, std::string> WebServer::GenerateLocaleDict() {
+	std::vector<std::string> translationKeys = GetTranslationKeys();
+    
+    map<std::string, std::string> localeMap;
+    for (auto const &translationKey : translationKeys) {
+		localeMap.insert(pair<std::string, std::string>(translationKey, _(translationKey)));
+	}
+    return localeMap;
+}
+
+std::vector<std::string> WebServer::GetTranslationKeys() {
+	std::vector<std::string> tranlationKeys = { 
+		"Performous web frontend",
+		"View database",
+		"View playlist",
+		"Search and Add",
+		"Sort by",
+		"Artist",
+		"Title",
+		"Language",
+		"Edition",
+		"Creator",
+		"Sort order",
+		"Normal",
+		"Inverted",
+		"Update every 10 sec",
+		"Refresh database",
+		"Upcoming songs",
+		"Refresh Playlist",
+		"Credits",
+		"Search",
+		"Available songs",
+		"Search for songs",
+		"Yes",
+		"No"
+	};
```

Comment:
One question how would one get this specific set of 'keys' then? since i need them within `webserver.cc` or else the webserver doesn't get to send the json dictionairy (with these keys)

Currently we're putting everything in a `map<key (string), translated_value(string)>` and convert this map to a jsontree which then is used clientsided.